pkg/validationfile/blocks/assertions.go

```diff
@@ -25,13 +29,16 @@ type Assertions struct {
 
 // Assertion is a parsed assertion.
 type Assertion struct {
-	// RelationshipString is the string form of the assertion.
-	RelationshipString string
+	// RelationshipWithContextString is the string form of the assertion.
+	RelationshipWithContextString string
```

Comment:
Because it is not a simple relationship anymore, but rather a string of the form `relationship with {context}` (if context is specified). I wanted to make that explicitly clear. I'll update the comment as well.